man/man8/zpool.8

```diff
@@ -1544,14 +1544,14 @@ Run a script (or scripts) on each vdev and include the output in zpool iostat
 .sp
 The \fB-c\fR option allows you to run script(s) for each vdev and display the
 output in zpool iostat.  For security reasons, a user can only execute scripts
-found in the /<etc>/zfs/zpool.d directory as an unprivileged user.  However, a
-privileged user can run \fB-c\fR if they have the ZPOOL_SCRIPTS_AS_ROOT
-environment variable set.  If a script requires the use of a privileged
-command (like smartctl) then it's recommended you allow the user access to it in
-/etc/sudoers.  For example, to allow user "zfsuser" access to "smartctl -a", add
-the following to /etc/sudoers:
+as an unprivileged user. A user may run a script from <home>/.zfs/zpool,
+/<etc>/zfs/zpool.d, or provide a full path to a script. A privileged user can
+run \fB-c\fR if they have the ZPOOL_SCRIPTS_AS_ROOT environment variable set.
+If a script requires the use of a privileged command (like smartctl) then it's
+recommended you allow the user access to it in /etc/sudoers.  For example, to
+allow user "zfsuser" access to "smartctl -a", add the following to /etc/sudoers:
 
-zfsuser ALL=NOPASSWD: /usr/sbin/smartctl -a /dev/sd[a-z]*, NOEXEC: /usr/sbin/smartctl -a /dev/sd[a-z]*`
+zfsuser ALL=NOPASSWD: /usr/sbin/smartctl -a /dev/sd[a-z0-9]*, NOEXEC: /usr/sbin/smartctl -a /dev/sd[a-z0-9]*`
```

Comment:
We should expand the pattern to match `/dev/hd` devices too, and maybe even `/dev/dm-[0-9]` devices which would include digits and dashes.